src/views/ProjectProfitability/ProjectProfitability.tsx

```diff
@@ -122,53 +125,14 @@ export const ProjectProfitabilityView = ({
         <Container name='project' ref={containerRef}>
           <>
             {activeTab === 'overview' && (
-              <div className='Layer__accounting-overview-profit-and-loss-charts'>
-                <Toggle
-                  name='pnl-detailed-charts'
-                  options={[
-                    {
-                      value: 'revenue',
-                      label: 'Revenue',
-                    },
-                    {
-                      value: 'expenses',
-                      label: 'Expenses',
-                    },
-                  ]}
-                  selected={pnlToggle}
-                  onChange={e =>
-                    setPnlToggle(e.target.value as PnlToggleOption)
-                  }
-                />
-                <Container
-                  name={classNames(
-                    'accounting-overview-profit-and-loss-chart',
-                    pnlToggle !== 'revenue' &&
-                      'accounting-overview-profit-and-loss-chart--hidden',
-                  )}
-                >
-                  <ProfitAndLoss.DetailedCharts
-                    scope='revenue'
-                    hideClose={true}
-                    // stringOverrides={stringOverrides?.profitAndLoss?.detailedCharts}
-                    // chartColorsList={chartColorsList}
-                  />
-                </Container>
-                <Container
-                  name={classNames(
-                    'accounting-overview-profit-and-loss-chart',
-                    pnlToggle !== 'expenses' &&
-                      'accounting-overview-profit-and-loss-chart--hidden',
-                  )}
-                >
-                  <ProfitAndLoss.DetailedCharts
-                    scope='expenses'
-                    hideClose={true}
-                    // stringOverrides={stringOverrides?.profitAndLoss?.detailedCharts}
-                    // chartColorsList={chartColorsList}
-                  />
-                </Container>
-              </div>
+              <AccountingOverview
+                stringOverrides={{}}
+                tagFilter={tagFilter ? tagFilter : undefined}
+                onTransactionsToReviewClick={() => {
+                  console.log('clicked')
+                }}
+                enableOnboarding={false}
+              />
```

Comment:
This works but getting the PNL report to work this way is pretty invasive. In my branch I just use the tagFilter field on the ProfitAndLoss component an dit works for both the ProfitAndLoss.DetailedCharts and ProfitAndLoss.Report

Reply:
That said I do like the way this looks better for the overview tab, so I suggest that we just go with my approach for the report and yours for the accounting overview